src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -811,7 +811,8 @@ keypairForSignature(Json::Value const& params, Json::Value& error)
             return {};
         }
 
-        if (secretType == jss::secret.c_str())
+        // using strcmp as pointers may not match (see  https://bit.ly/3OGvf0E)
```

Comment:
In general, I recommend using the long link for long-lasting documentation. I treat short links as temporarily-convenient throwaways.

Reply:
done